### PR TITLE
GitHub actions for NESTML

### PR DESCRIPTION
**Pull request by pnbabu**
- Migrate content from `.travis.yml` to `.github/workflows/nestml-build.yml`
- Remove `.travis.yml` file
- Successful build link: https://github.com/pnbabu/nestml/actions/runs/571667802

### CODE REVIEW

**Review by jougs — APPROVED** [review]
Amazing work! Many thanks! I hereby approve, but leave the final decision to @clinssen.

File: .github/workflows/nestml-build.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        run: |
+          wget http://www.antlr.org/download/antlr-4.8-complete.jar
+          echo \#\!/bin/bash > antlr4
+          echo java -cp \"`pwd`/antlr-4.8-complete.jar:$CLASSPATH\" org.antlr.v4.Tool \"\$@\" >> antlr4
+          echo >> antlr4
+          chmod +x antlr4
+          echo PATH=$PATH:`pwd` >> $GITHUB_ENV
+
+      # Install Python dependencies
+      - name: Python dependecies
```
Comment (clinssen) [inline .github/workflows/nestml-build.yml:45]:
```suggestion
      - name: Python dependencies
```

File: .github/workflows/nestml-build.yml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+      # Tests
+      - name: Run unit tests
+        run: |
+          pytest -s -o norecursedirs='*'  -o log_cli=true -o log_cli_level="DEBUG" tests
+
+      # Install NEST simulator
+      - name: NEST simulator
+        run: |
+          python -m pip install cython pytest-xdist
```
Comment (clinssen) [inline .github/workflows/nestml-build.yml:81]:
```suggestion
          python -m pip install cython
```
I believe this is no longer necessary.

**Review by clinssen — CHANGES_REQUESTED** [review]
Many thanks for this! Just a few comments.

The "Install NESTML" stage appears twice in slightly different form, is this necessary?

**Comment by clinssen** [comment]
One more suggestion. Could we use the same versions for Antlr (in Java) as for the Python Antlr4 runtime?  Right now the former is at 4.8 and the latter is at 4.9.1. This results in a lot of warning messages in the log. Either we can ==4.8 in requirements.txt, or you'll have to update the links to the Antlr .jar download. Cheers!

**Comment by pnbabu** [comment]
> One more suggestion. Could we use the same versions for Antlr (in Java) as for the Python Antlr4 runtime? Right now the former is at 4.8 and the latter is at 4.9.1. This results in a lot of warning messages in the log. Either we can ==4.8 in requirements.txt, or you'll have to update the links to the Antlr .jar download. Cheers!

Yes, changed the version of the jar to 4.9.1

**Review by clinssen — APPROVED** [review]
Don't know why the removed .travis.yml is giving a merge conflict—if you could just fix that, I'm A-OK with the changes. Really great to have this, many thanks!

**Review by clinssen — APPROVED** [review]

**Pull request merged**